src/otp/migrations/mod.rs

```diff
@@ -1,27 +1,25 @@
 use super::otp_element::OTPDatabase;
 struct Migration<'a> {
     to_version: u16, // Database version which we are migrating on
-    migration_function: &'a dyn Fn(&mut OTPDatabase) -> color_eyre::Result<()>, // Function to execute the migration
+    migration_function: &'a dyn Fn(&mut OTPDatabase), // Function to execute the migration
```

Comment:
Please revert this, because in the future migrations could lead to errors, so it is better to keep this signature.